cppcache/integration-test/testLRUList.cpp

```diff
@@ -29,7 +29,7 @@ END_TEST(NotOnWindows)
 
 //#define BUILD_CPPCACHE
 
-#include <LRUList.cpp>
+#include <LRUList.hpp>
```

Comment:
Haha.

Reply:
GEODE-6844

This is a much simpler PR than the last attempt, I hope you'll agree.